session/session_test.go

```diff
@@ -164,6 +164,7 @@ func (s *testSessionSuite) TestErrorRollback(c *C) {
 		go func() {
 			defer wg.Done()
 			localTk := testkit.NewTestKitWithInit(c, s.store)
+			localTk.MustExec("set @@session.tidb_retry_limit = 9999999")
```

Comment:
This is a bad idea, why we need to retry so many times?

Reply:
I guess it also confuses other people, which retry count is expected? Can it work if we set it to a smaller value?